src/create-config.ts

```diff
@@ -76,7 +76,7 @@ const buildNodeSettings = (
     },
     logFormat: 'plain' as const,
     logLevel: 'INFO' as const,
-    stage: `prod-${timestamp}`,
+    stage: `prod${timestamp}`,
```

Comment:
Is this intentional? Do you mind sharing the requirement for this change?

Reply:
there was issue which burak bought up about gcp having issues when stage has a '-' in it

Reply:
This is not a thing, please revert. I said "changing `stage`, for example, by adding a `-` at the end fixes the problem". It's not about the symbol specifically.

Reply:
got it